transit_odp/browse/templates/browse/guideme/use_api.html

```diff
@@ -37,6 +37,16 @@
 
 <p class="govuk-body">
     {% blocktrans %}
-    Helpful cases studies are provided to inspire and support your API data journey on BODS.
+        Helpful cases studies are provided to inspire and support your API data journey on BODS.
+    {% endblocktrans %}
+</p>
+
+<a class="govuk-link govuk-!-font-size-19" href="https://github.com/department-for-transport-BODS/bods-data-extractor">
```

Comment:
Typically we add `target="_blank"` to the a-tags to make the link open in another tab for outside links.